sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -45,16 +45,22 @@ function throwMissingEnvironmentVariablesError(envVars: EnvVarKeys[]): void {
   });
 }
 
-function getEnvVarValue(name: string): string | undefined {
+function getEnvVarValue(name: string, forBrowser?: boolean): string | undefined {
+  let result;
   if (isNode) {
-    return process.env[name];
+    result = process.env[name];
   } else {
     // @ts-ignore
-    return window.__env__[name];
+    result = window.__env__[name];
   }
+
+  if (forBrowser) {
+    result = result + "_BROWSER";
```

Comment:
Are we expecting env variable names to have the `_BROWSER` prefix or their values to have this suffix?

Reply:
Both the names and values would have the 'browser' suffix.
This here was needed for the name - please refer to latest changes

Reply:
Should we do this appending of `"_BROWSER"` on the `name` instead of the `result`?

Reply:
Oops, fixed this in the script, but not in here.
Just fixed it - got missed while copy pasting.
(It's going to be a little problematic to maintain duplicated code :-/)